backend/sftp/sftp.go

```diff
@@ -28,6 +28,8 @@ type SFTP struct {
 	cmd *exec.Cmd
 }
 
+var sysProcAttr syscall.SysProcAttr
```

Comment:
This solution works, but I don't like the added global state. What about having a function (like for the `Chmod()` above) that sets the right options? That's much clearer and you can directly see where the options in `cmd.SysProcAttr` come from, instead of having to dig through `init()` methods...